client/powerline.c

```diff
@@ -42,7 +42,7 @@ void do_write(int sd, const char *raw, size_t len) {
 	}
 }
 
-inline size_t true_sun_len(const struct sockaddr_un *ptr) {
```

Comment:
What’s the problem with `inline`? And does not this work with `static inline`? This looks like a gcc bug so far.